### PR TITLE
Kirilushkin Aleksandr Tasks 61-70

### PR DESCRIPTION
**Pull request by KirilushkinAleksandr**

### CODE REVIEW

File: tasks/61_spinal-case/spinal-case.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+function spinalCase(str) {
+  const separators = [' ', '_'];
+  const arr = str.split('');
```
Comment (alexkond) [inline tasks/61_spinal-case/spinal-case.js]: You're changing every value of array so it looks like it is a case for `.map()` method.

File: tasks/62_swap-cases/swap-cases.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+function swapCases(input) {
+  const res = input.split('');
+  return res.map(
+      function(item) {
```
Comment (alexkond) [inline tasks/62_swap-cases/swap-cases.js]: Consider using arrow function

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+function smallestCommons(arr) {
+    let start = arr[0] < arr[1] ? arr[0] : arr[1];
```
Comment (alexkond) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: Consider using `Math.max()` here

File: tasks/67_find-elements/find-elements.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+function findElement(arr, func) {
+  const resIndex = arr.findIndex(func);
```
Comment (alexkond) [inline tasks/67_find-elements/find-elements.js]: I think the idea of this task is to write this method yourself, not to create a wrapper.

**Review by alexkond — CHANGES_REQUESTED** [review]

File: tasks/65_smaller-common-multiple/smaller-common-multiple.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ * smallestCommons([1, 5])
+ * function smallestCommons finds the smallest common multiple
+ *  of the provided parameters that can be evenly divided by both,
+ * as well as by all sequential numbers in the range between these parameters
+ * @param {Array} arr, array of two numbers, lower and upper bounds of range
+ * @return {Number} number that can be evenly divided by both elements of array
+ */
+function smallestCommons(arr) {
+  let start = Math.min(arr[0], arr[1]);
+  let finish = Math.max(arr[0], arr[1]);
```
Comment (vovcyan) [inline tasks/65_smaller-common-multiple/smaller-common-multiple.js]: const

**Review by vovcyan — DISMISSED** [review]

**Review by alexkond — APPROVED** [review]

**Review by savchukm — APPROVED** [review]

**Pull request merged**